app/assets/javascripts/turbo.js

```diff
@@ -1269,6 +1286,9 @@ class StreamElement extends HTMLElement {
   get target() {
     return this.getAttribute("target");
   }
+  get replaceIfPresent(){
```

Comment:
The codebase's copy of Turbo comes from [hotwired/turbo](https://github.com/hotwired/turbo/), so any JavaScript changes should be proposed there.

With that being said, this PR seems to share some overlap with https://github.com/hotwired/turbo/pull/191.

The [guidance provided in that PR](https://github.com/hotwired/turbo/pull/191#pullrequestreview-602190778) was to handle it as part of the `broadcast_{append,prepend}_to` suite of helpers. 

You're proposing something similar here. Instead of making `replace_if_present:` an option, have you tried combining `broadcast_remove` with the append and prepend broadcasts?

Reply:
Sorry I didn't look into the turbo repo :-/ 

It looks like what you did in hotwired/turbo#191 would do this, whereas my version was opt-in.
But indeed, the end effect would be the same as doing a remove/append in the templates.

I will close this, thanks for the heads up !

Reply:
@jeromecornet No worries! I don't think https://github.com/hotwired/turbo/pull/191 is viable, but I think your proposed changes are. Would you be interested in transforming your proposal to incorporate the remove+append style of broadcast?

Reply:
ok, reopening to see where it can go.... 

I reverted the changes to turbo.js and replaced the turbo tag generation with a remove + append in https://github.com/hotwired/turbo-rails/pull/153/commits/e8aa95f9165636fd69778905f352e89f9766ee2e